demo-extended/src/main/java/com/novoda/simplechromecustomtabs/demo/ExtendedDemoActivity.java

```diff
@@ -26,7 +26,7 @@
 
     private static final Uri WEB_URL = Uri.parse("http://www.novoda.com");
     private static final int REQUEST_CODE_VIEW_SOURCE = 1;
-    private static final int REQUEST_CODE_NOVODA_LONDON = 2;
+    private static final int REQUEST_CODE_SHARE_URL = 2;
```

Comment:
👍 Yep, let's not be selfish and place an action which I think representes better a very common action that one would like to perform.